.gitignore

```diff
@@ -46,13 +46,12 @@ build-iPhoneSimulator/
 
 # for a library or gem, you might want to ignore these files since the code is
 # intended to run in multiple environments; otherwise, check them in:
```

Comment:
removed `.ruby-version` from vcs since this comment suggested it and all of the other ruby gems i checked do not include it